scripts/generate_scripts_functions.py

```diff
@@ -214,10 +214,10 @@ def arg_type(x, el):
     if 'AllowGlobalVar' in el.attrib:
         allowGlobal = True if el.attrib['AllowGlobalVar'] == 'true' else False
     if allowLocal == False and allowGlobal == True:
-        outRef = 'Global'
+        outRef = '&'
         outName = 'arg' + str(x) + 'G'
     elif allowLocal == True and allowGlobal == False:
-        outRef = 'Local'
+        outRef = '&'
```

Comment:
Sorry to bother about this again, this is a visible ref now, but now we loose the info wether it's global or local?
Are we certain we don't need it?

(Am I overthinking this?)